izumi-reflect/izumi-reflect/src/main/scala_3/izumi/reflect/dottyreflection/Inspector.scala

```diff
@@ -91,6 +91,10 @@ abstract class Inspector(protected val shift: Int) extends InspectorBase {
         log(s"TYPEREPR UNSUPPORTED: LazyRef occured $lazyref")
         NameReference("???")
 
+      // Matches CachedRefinedType for this ZIO issue https://github.com/zio/zio/issues/6071
+      case ref: Refined { def parent: TypeRepr } =>
```

Comment:
@kitlangton Refined is a TypeTree, not a TypeRepr so this match can never match - except that it's skipped due to additional refinement. Actually the reflective access seems unnecessary - the method is public in `RefinementMethods`, so it should work with
```suggestion
      case ref: Refinement =>
```

Reply:
Oops! Good catch :)